src/sessionOne/BasicTypes.kt

```diff
@@ -1,2 +1,168 @@
+ * The main method is the entry point for any application in almost all
+ * programming languages including kotlin, write all your executable code here.
+ * NB : Comments represent non-compilable and non-executable code.
+ */
+fun main() {
+    // Comments represent non-compilable and non-executable code
+    //single line comment
+    /**
+     * Documentation comment
+     */
+
+    /*
+     * Multiline comment
+     * Yes, as you can see, this is not compiled by the kotlin compiler
+     */
+
+    // printing things onto your screen
+    println("Welcome to our lovely course !")
+
+    // basic types
+
+    // an integer is either a +ve or -ve non-fractional 32-bit number
+    val positiveInt = 23222222
+    val integerSize = Int.SIZE_BITS
+    val integerMax: Int = Int.MAX_VALUE
+    println(positiveInt)
+    println(integerSize)
+    println(integerMax)
+
+    // a long is either a +ve or -ve non-fractional 64-bit number
+    val positiveLong = 23222222L
+    val longSize = Long.SIZE_BITS
+    val longMaxValue: Long = Long.MAX_VALUE
+    println(positiveLong)
+    println(longSize)
+    println(longMaxValue)
+
+    // unsigned integers are either +ve nor -ve ones
+    var unsignedInt : UInt = 20u
+    unsignedInt += 2u
+    println(unsignedInt)
+
+    // boolean values are either true or false
+    var myBoolean = true
+    // print before flipping
+    println("Boolean test : $myBoolean")
+    // to flip a boolean use the inverse or inverter operator
+    myBoolean = !myBoolean
+    // print after flipping
+    println("Boolean test : $myBoolean")
+
+
+    // if we want to represent a letter from alphabet
+    var letter = 'A'
+    letter = 'a'
+    println("My Letter is a changing value : $letter")
+
+    // using a value not a variable
+    val letter2 = 'D'
+    println("My Letter is a constant value : $letter2")
+
+    //-------------------------Exercise-------------------------
+
+    // 1) creating your ID data
+    // your name is a constant and its a String or some characters from alphabets
+    val myName = "Mohammed"
+    // your age is changing
+    var myAge = 21
+    // your work is a constant
+    // defining the type of data as String is optional here
+    val work: String = "Engineer"
+    // your religion is a constant
+    val religion = "Muslim"
+    // your ID number is stable
+    // defining the type of data is a must here or else append L at the end of a big number
+    val idNumber: Long = 22341512312343
+    // now print your own ID
+    println("Name : $myName")
+    println("Age : $myAge")
+    println("Work : $work")
+    println("Religion : $religion")
+    println("ID Number : $idNumber")
+
+    // 2) print your ID after 2 years
+    println("Name : $myName")
+    println("Age : ${myAge + 2}")
+    println("Work : $work")
+    println("Religion : $religion")
+    println("ID Number : $idNumber")
+
+    // Basic Operations
+
+    // 1) a pointer (mouse) and a screen size, showing addition/subtraction operations
```

Comment:
keep it simpller here like calc app or just simple operations

Reply:
@Ahmedshehatah Should I remove the pointer and the screen example ? Or just add a simple calculator ?

Reply:
Or should we move it to the exercise section at the end of the code ?

Reply:
yeah remove them and add a smiple calc